tests/Cache/CacheClientTest.php

```diff
@@ -768,4 +805,129 @@ public function testListErase_MissingList()
+
+        $otherField = uniqid();
+        $response = $this->client->dictionaryGet($this->TEST_CACHE_NAME, $dictionaryName, $otherField);
+        $this->assertNotNull($response->asMiss());
+    }
+
+    public function testDictionaryNoRefreshTtl()
+    {
+        $dictionaryName = uniqid();
+        $field = uniqid();
+        $value = uniqid();
+        $response = $this->client->dictionarySet($this->TEST_CACHE_NAME, $dictionaryName, $field, $value, false, 5);
+        $this->assertNotNull($response->asSuccess());
+        sleep(1);
+
+        $response = $this->client->dictionarySet($this->TEST_CACHE_NAME, $dictionaryName, $field, $value, false, 10);
+        $this->assertNotNull($response->asSuccess());
+        sleep(4);
+
+        $response = $this->client->dictionaryGet($this->TEST_CACHE_NAME, $dictionaryName, $field);
+        $this->assertNotNull($response->asMiss());
+    }
+
+    public function testDictionaryThrowExceptionForEmptyDictionaryName()
+    {
+        $dictionaryName = "";
+        $field = uniqid();
+        $response = $this->client->dictionaryGet($this->TEST_CACHE_NAME, $dictionaryName, $field);
+        $this->assertNotNull($response->asError());
+        $this->assertEquals(MomentoErrorCode::INVALID_ARGUMENT_ERROR, $response->asError()->errorCode());
+    }
+
+    public function testDictionaryThrowExceptionForEmptyFieldName()
+    {
+        $dictionaryName = uniqid();
+        $field = "";
+        $response = $this->client->dictionaryGet($this->TEST_CACHE_NAME, $dictionaryName, $field);
+        $this->assertNotNull($response->asError());
+        $this->assertEquals(MomentoErrorCode::INVALID_ARGUMENT_ERROR, $response->asError()->errorCode());
+    }
+
+    public function testDictionaryThrowExceptionForEmptyValueName()
+    {
+        $dictionaryName = uniqid();
+        $field = uniqid();
+        $value = "";
+        $response = $this->client->dictionarySet($this->TEST_CACHE_NAME, $dictionaryName, $field, $value, false);
+        $this->assertNotNull($response->asError());
+        $this->assertEquals(MomentoErrorCode::INVALID_ARGUMENT_ERROR, $response->asError()->errorCode());
+    }
+
+    public function testDictionaryRefreshTtl()
+    {
+        $dictionaryName = uniqid();
+        $field = uniqid();
+        $value = uniqid();
+        $response = $this->client->dictionarySet($this->TEST_CACHE_NAME, $dictionaryName, $field, $value, false, 2);
+        $this->assertNotNull($response->asSuccess());
+        $response = $this->client->dictionarySet($this->TEST_CACHE_NAME, $dictionaryName, $field, $value, true, 10);
+        $this->assertNotNull($response->asSuccess());
+        sleep(2);
+
+        $response = $this->client->dictionaryGet($this->TEST_CACHE_NAME, $dictionaryName, $field);
+        $this->assertNotNull($response->asHit());
+        $this->assertEquals($value, $response->asHit()->value());
+    }
+
+    public function testDictionaryDeleteThrowExceptionForEmptyCacheName()
```

Comment:
function name is now out of date -- returns error instead of throws exception :)